index.Rmd

```diff
@@ -357,14 +371,13 @@ git config --global user.email you@example.com
 
 10. *Make changes*
 
-To see Git in action, you need to make some changes in your repository. Try it by creating a new file in the directory where you cloned your new project.
-You can do that using the *Text editor* (gedit), or from the terminal using the `touch` command.
+To see Git in action, you need to make some changes in your repository. Try it by creating a new file in the directory where you cloned your new project. You can do that using the *Text editor* (gedit), or from the terminal using the `touch` and `nano` command.
 
-Once you are done, go back to Git Cola. If you closed the window, you can get back to your repository by launching Git Cola and clicking on its path in the *list*. You will see some changes:
+Let's go back to Git Cola. If you closed the previous Git Cola window, you can get back to your repository by launching Git Cola and clicking on its path in the *list*. You will see some changes:
 
 ![Changes pending in Git Cola](figs/git-cola-changes.png)
 
-To see a list of files with pending changes from the command line, use `git status` while in a git repository. To see what exactly changed in each of these files, use `git diff`. For example:
+To see a list of files with pending changes from the command line, use `git status` while in a git repository folder. To see what exactly changed in each of these files, use `git diff`. For example:
```

Comment:
Technically "directory", "folder" is a Windows term (which incidentally means "flyer" in Dutch ;) )